Dockerfile

```diff
@@ -1,14 +1,23 @@
-FROM python:3.7-stretch
+FROM python:3.7
 
-RUN apt-get update && mkdir -p /app
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  python3.7 \ 
+  python3-pip \
+  python3-flask
+#  python3.7-venv
```

Comment:
I python is available in the image. It is built from the sources. If we want to use the packaged version then we should use a different base image.